packages/cli/src/commands/setup.js

```diff
@@ -4,7 +4,7 @@ import terminalLink from 'terminal-link'
 
 export const builder = (yargs) =>
   yargs
-    .commandDir('./setup', { recurse: true })
+    .commandDir('./setup', { recurse: true, exclude: /ui\/.*\.js$/ })
```

Comment:
This would've fixed the issue I ran into (unless someone's project is actually named `ui`)
```suggestion
    .commandDir('./setup', { recurse: true, exclude: /\/ui\/.*\.js$/ })
```